src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -592,7 +600,8 @@ private StudentEnrollDetails enrollStudent(StudentAttributes validStudentAttribu
         return enrollmentDetails;
     }
     
-    /* All empty lines or lines with only white spaces will be skipped.
+    /*
+     * All empty lines or lines with only white spaces will be skipped.
      * The invalidity info returned are in HTML format.
```

Comment:
This is a block comment not a Doc comment. Block comments start with /* where as Doc comments start with /**